engines/python/setup/djl_python/utils.py

```diff
@@ -147,3 +144,9 @@ def _fetch_adapters_from_input(input_map: dict, inputs: Input):
         adapters_per_item = [adapters_per_item]
 
     return adapters_per_item
+
+
+def _validate_adapters(adapters_per_item, adapter_registry):
+    for adapter_name in adapters_per_item:
+        if adapter_name not in adapter_registry and adapter_name != "":
```

Comment:
Does this also need to handle `None`? I'm not sure how this works.

Maybe this is better, so it handles None and "" and any other falsy values: `if adapter_name and adapter_name not in adapter_registry:`

Reply:
I don't think `adapter` will be `None` but sure this is better, updated.